pkgs/tools/security/rbw/default.nix

```diff
@@ -49,6 +51,7 @@ rustPlatform.buildRustPackage rec {
 
   postInstall = ''
     install -Dm755 -t $out/bin bin/git-credential-rbw
+    patchShebangs $out/bin/git-credential-rbw
```

Comment:
I believe this is not needed, since `patchShebangsAuto` adds it by default by default
```suggestion
```

Reply:
No. Without this line it's /bin/sh instead of bash. You can test.